ecs_fargate/assets/service_checks.json

```diff
@@ -2,13 +2,13 @@
     {
         "agent_version": "6.0.0",
         "integration": "Amazon Fargate",
-        "check": "ecs_fargate.can_connect",
+        "check": "fargate_check",
```

Comment:
Hm is the service check only showing `fargate_check` with no namespacing? I think that we should probably update the code for this instead, since most other integrations have a `*.can_connect` service check.

Reply:
Actually changing the service check name would be a breaking change, so it would be too much of a hassle to standardize this particular service check.